src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -5389,7 +5390,7 @@ public static double Calculate(double[] f)
 5180801");
         }
 
-        [ConditionalFact(typeof(ClrOnly), typeof(NoIOperationValidation), Reason = "https://github.com/dotnet/roslyn/issues/29428")]
+        [ConditionalFact(typeof(ClrOnly), typeof(NoIOperationValidation), Reason = "https://github.com/dotnet/roslyn/issues/29428", AlwaysSkip = "https://github.com/dotnet/roslyn/issues/46361")]
```

Comment:
>https://github.com/dotnet/roslyn/issues/46361 [](start = 146, length = 45)

Do other branches experience the same issue? Otherwise, we should treat this failure as possibly related to the changes around covariant returns (perhaps even in this PR), i.e. compiler is possibly consuming more stack than before, which brings this test close to the failure point.

Reply:
Yes, this test is flaky on other branches.  It doesn't run the test on a fresh thread.

---
In reply to: [461776977](https://github.com/dotnet/roslyn/pull/46367#discussion_r461776977) [](ancestors = 461776977)